typeDefs.js

```diff
@@ -198,7 +198,7 @@ const typeDefs = gql`
     registerUser(user: UserInput!): User
     loginUser(user: UserInput!): User
     deleteUser(id: ID!): User
-    updateUser(id: ID!, user: UserInput!): User
+    updateUser(user: UserInput!): User
```

Comment:
We need two separate mutations for updating the user by himself and updating the user by admin. The second one will require having id

Reply:
fixed